scripts/build-dist

```diff
@@ -40,10 +40,12 @@ EOF
 python setup.py sdist --dist-dir "$BUILD_DIR"
 
 # Build x86_64 linux and manylinux wheels
-docker run -it --rm -v "${BUILD_DIR}:/dd-trace-py" -e "ARCH=x86_64" quay.io/pypa/manylinux1_x86_64 /bin/bash -c "${build_script}"
+docker pull quay.io/pypa/manylinux1_x86_64
+docker run -it --rm -v "${BUILD_DIR}:/dd-trace-py" -e "ARCH=x86_64" quay.io/pypa/manylinux1_x86_64:latest /bin/bash -c "${build_script}"
 
 # Build i686 linux and manylinux wheels
-docker run -it --rm -v "${BUILD_DIR}:/dd-trace-py" -e "ARCH=i686" quay.io/pypa/manylinux1_i686 linux32 /bin/bash -c "${build_script}"
+docker pull quay.io/pypa/manylinux1_i686
+docker run -it --rm -v "${BUILD_DIR}:/dd-trace-py" -e "ARCH=i686" quay.io/pypa/manylinux1_i686:latest linux32 /bin/bash -c "${build_script}"
```

Comment:
```suggestion
docker pull quay.io/pypa/manylinux1_i686:latest
docker run -it --rm -v "${BUILD_DIR}:/dd-trace-py" -e "ARCH=i686" quay.io/pypa/manylinux1_i686:latest linux32 /bin/bash -c "${build_script}"
```

Reply:
?

Reply:
`docker pull` defaults to the `:latest` tag so adding it here would be redundant

Reply:
👍